pag_nodes.py

```diff
@@ -61,7 +79,7 @@ def post_cfg_function(args):
                 if signal_scale < 0:
                     signal_scale = 0
 
-            if signal_scale == 0:
+            if signal_scale == 0 or check_skip(sigma[0],sigma_start,sigma_end):

```

Comment:
`check_skip` is probably incorrect.
When `sigma_start_percentage=100.0` and `sigma_end_percentage=0.0`, `check_skip` returns True for the first step of `euler` sampler (sigma[0] is 14.6146, and sigma_start is 14.5855)

Reply:
It's also incompatible with some samplers from https://github.com/Koishi-Star/Euler-Smea-Dyn-Sampler (since they work with sigmas outside of sigma_max range)

I think this functionality either should operate on absolute sigma values (this would fix problems, but would be inconvenient to use) or should be moved into a different node (leaving the old version as it is for compatibility)

Reply:
Ouch, I see! Thank you for pointing it out. I will seek a better solution in the upcoming days.